.github/ISSUE_TEMPLATE/static_data.yml

```diff
@@ -0,0 +1,134 @@
+name: Static Data Update
+description: Request static data be added or updated
+labels: ["Static Data Mgmt"]
+assignees:
+  - KateFriedman-NOAA
+  - WalterKolczynski-NOAA
+
+body:
+  - type: dropdown
+    attributes:
+      label: Type of static data
+      description: |
+        - Fix: large static files needed to run global-workflow
+        - Initial conditions: ICs needed to run one of global-workflow's included tests
+        - Experimental data: Temporary datasets needed for development that will be replaced by other sources for operations (e.g. data that will be in DCOM)
+      options:
+        - Fix
+        - Initial conditions
+        - Experimental data
+    validations:
+      required: true
+
+  - type: dropdown
+    attributes:
+      label: Target directory
+      options:
+        - "Fix: aer"
+        - "Fix: am"
+        - "Fix: chem"
+        - "Fix: cice"
+        - "Fix: cpl"
+        - "Fix: datm"
+        - "Fix: gdas/crtm"
+        - "Fix: gdas/fv3jedi"
+        - "Fix: gdas/gsibec"
+        - "Fix: gldas"
+        - "Fix: glwu"
+        - "Fix: gsi"
+        - "Fix: lut"
+        - "Fix: mom6"
+        - "Fix: orog"
+        - "Fix: raw"
+        - "Fix: reg2grb2"
+        - "Fix: sfc_climo"
+        - "Fix: ugwd"
+        - "Fix: verif"
+        - "Fix: wave"
+        - "Fix: *NEW*"
+        - "ICs: C48/C48 mx500"
+        - "ICs: C96/C48"
+        - "ICs: C96 mx100"
+        - "ICs: C384/C192"
+        - "ICs: C384 mx025"
+        - "ICs: C768 mx025"
+        - "ICs: C1152 mx025"
+        - "ICs: *NEW RESOLUTION*"
+        - "Exp Data: GOCART emissions"
```

Comment:
```suggestion
        - "Exp Data: Global Dump Archive - new or experimental (RUNx.PDY)"
        - "Exp Data: GOCART emissions"
```

Reply:
I originally had a dump requests as a separate category, but the info we need for that is different enough from the others it just made descriptions messy. I meant to make a new template for it, but forgot. Will do that in a follow-up PR.

Reply:
Okie dokie, sounds good!